docker/appserver/Tomcat/example/Dockerfile

```diff
@@ -8,4 +8,5 @@ COPY target/dependencies/*.jar /usr/local/tomcat/lib/
 COPY src/scripts/context.xml /usr/local/tomcat/conf/Catalina/localhost/iaf.xml
 
 # copy configuration, properties and testscenarios
-COPY target/dependencies/frank /opt/frank
+COPY target/dependencies/frank/resources target/dependencies/frank/configuration /usr/local/tomcat/webapps/iaf/WEB-INF/classes/
```

Comment:
In plaats van naar uitgepakte war te kopieren, kan je ook /opt/frank/resources en /opt/frank/configuration aan het classpath toevoegen.

Reply:
Teruggedraaid, had niet door dat het met op het classpath zetten het ook kon, bij een eerste test had die resources niet gepakt maar lag aan het classpath dat niet helemaal goed stond.